src/pages/Components/Select/index.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
this file should be handled by designers, so we don't have conflicts. 
Previews can be added at the pooling phase. 
Plus, we refactored all the inputs to use only one preview for all, and we removed pointer events states because it can be viewed by the user moving the cursor, and does not add more 'usage' information.